### PR TITLE
Add repo templates and elixir coveralls guide

### PR DESCRIPTION
**Pull request by fabriziosestito**
This PR adds:
- CONTRIBUTING.md template
- badges.md template
- elixir coveralls guide
- elixir ci coveralls step

### CODE REVIEW

File: templates/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+If you are reporting a `bug`, please share a file generated using the
+`trento-support.sh` script with the following params:
+
+```
+# trento-support.sh --collect all --output file-tgz
+```
+
+and include the output to your issue. The script should remove sensitive data
+automatically but please make sure you are not sharing anything sensible yourself.
```
Comment (jagabomb) [inline templates/CONTRIBUTING.md:23]: ..but please make sure you are not sharing **any sensitive data of your own**.
(Suggestion highlighted in bold)

File: templates/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Submitting changes
+
+### Pull Requests guideline
+
+Reviews are hard. These few points will help reduce the effort and allow us to
```
Comment (jagabomb) [inline templates/CONTRIBUTING.md:29]: These few points will help **us to reduce the time and effort required** and allow us to merge your changes faster.
(Suggestion highlighted in bold)

File: templates/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
```
[the remark this answers is not in the document]
Comment (jagabomb) [reply templates/CONTRIBUTING.md:2]: @fabriziosestito I have added a couple of improvements. I would also suggest that @jamie-suse has a read through to make sure that the grammar is decent from a technical point of view.

File: coding-standards/elixir.md [hunk 4]
```diff
@@ -43,6 +43,46 @@ Testing guidelines:
...
 - [Controller tests as integration tests](https://groups.google.com/g/elixir-ecto/c/BKpLf092dWs/m/VaCvfZpEBQAJ) (_pre-Phoenix Context_ but still relevant)
 
+### Test coverage
+
+Test coverage is enforced by [coveralls.io](https://coveralls.io/). Please add [excoveralls](https://github.com/parroty/excoveralls) to the list of dependencies and setup `mix.exs` file as shown here:
+
+```elixir
+def project do
+  [
+    app: :excoveralls,
```
Comment (dottorblaster) [inline coding-standards/elixir.md:53]: If this is meant to be an example maybe I'd call this app `:excoveralls_example` or `:excoveralls_demo`

**Review by dottorblaster — APPROVED** [review]
LGTM

File: templates/CONTRIBUTING.md (hunk 1 above)
Comment (fabriziosestito) [reply templates/CONTRIBUTING.md:23]: @jagabomb thanks, addressed

File: templates/CONTRIBUTING.md (hunk 2 above)
Comment (fabriziosestito) [reply templates/CONTRIBUTING.md:29]: @jagabomb thanks!

File: templates/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+Thanks for showing interest and sharing your time to contribute to this project!
+
+This guide is meant to be used as a general guideline for creating issues or
+pull requests. We encourage all first contributors to give this a read to avoid
```
Comment (jamie-suse) [inline templates/CONTRIBUTING.md:6]: >first contributors

_first-time contributors_ or _new contributors_

File: templates/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+the issue as per our [labels](https://github.com/trento-project/projectname/labels)
+
+If you are reporting a `bug`, please share a file generated using the
+`trento-support.sh` script with the following params:
+
+```
+# trento-support.sh --collect all --output file-tgz
+```
+
+and include the output to your issue. The script should remove sensitive data
```
Comment (jamie-suse) [inline templates/CONTRIBUTING.md:22]: >and include the output *in* your issue.

**Pull request merged**